### ISSUE TITLE
not working in ubuntu focal

### ISSUE THREAD

**Issue by jpritikin** [jethrokuan/fzf#145]
I upgraded to focal and Ctrl-R doesn't work anymore. How to troubleshoot?

**Comment by jorgebucaran** [comment]
You could start by sharing more details.

**Comment by jpritikin** [comment]
Which details would be most helpful?

**Comment by 2m** [comment]
Are you using fish? If so a bug in fish 3.1.1 might have caused that: https://github.com/jethrokuan/fzf/issues/147#issuecomment-621001799

**Comment by jpritikin** [comment]
Yes, fish 3.1.0. I upgraded to fish 3.1.2 via the [ubuntu ppa](https://launchpad.net/~fish-shell/+archive/ubuntu/release-3/). I rebooted. Unfortunately, Cntl-R still does nothing. Anything else I can try?
